fibonacci/py/code.py

```diff
@@ -7,8 +7,11 @@ def fibonacci(n):
     return 1
   return fibonacci(n-1) + fibonacci(n-2)
 
-u = int(sys.argv[1])
-r = 0
-for i in range(u):
-  r += fibonacci(i)
-print(r)
+def main():
+    u = int(sys.argv[1])
+    r = 0
```

Comment:
Pythonic way here is to use builtin `sum` function. And type hints would be nice to have.

Reply:
The idea is to implement the "same" way across all implementations, so will not be adding sum at this time.